lib/server.js

```diff
@@ -23,6 +23,8 @@ app.use(morgan('combined', { skip: req => !util.isIndex(req.path) }));
 
 // proxy feed requests
 app.get('/proxy', function proxyFeed(req, res) {
+  res.setHeader('Max-Age', '30');
```

Comment:
Shouldn't this actually be `setHeader('Cache-Control', 'public, max-age=30')`?  Or does express translate those automagically or something?